apps/chat/src/store/publication/publication.reducers.ts

```diff
@@ -174,6 +172,18 @@ export const publicationSlice = createSlice({
     uploadRulesFail: (state) => {
       state.isRulesLoading = false;
     },
+    selectItemsToPublish: (
+      state,
+      { payload }: PayloadAction<{ ids: string[] }>,
+    ) => {
+      state.selectedItemsToPublish = union(
+        difference(state.selectedItemsToPublish, payload.ids),
+        difference(payload.ids, state.selectedItemsToPublish),
+      );
```

Comment:
just use `xor` instead of 2 `difference` and `union`

Reply:
it's different operations. It won't work with xor